docs/install/remote.md

```diff
@@ -50,67 +50,47 @@ Before you begin, ensure that the following prerequisites are met:
       To install all extras, run:<br>
       ```pip install mlrun[complete]```
      
-2. Alternatively, if you already installed a previous version of MLRun, upgrade it by running:
+3. Alternatively, if you already installed a previous version of MLRun, upgrade it by running:
 
     ```sh
     pip install -U mlrun==<version>
     ```
 
-2. Ensure that you have remote access to your MLRun service (i.e., to the service URL on the remote Kubernetes cluster).
+4. Ensure that you have remote access to your MLRun service (i.e., to the service URL on the remote Kubernetes cluster).
 
 ## Configure remote environment
+You have few options to configure your remote environment:
+1. Using the default `.env` file (**Recommended**)
```

Comment:
- Using the default `.env` file (**Recommended**)
- Using your own environment file
- Using MLRun SDK or CLI
- Using your IDE (e.g PyCharm or VSCode)